domains/index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: steps
+title: Domains
+has_children: false
+permalink: /domains/
+image: false
+title: "Domains"
+subhead: "Manage all custom domains on your site"
+context:
+  - title: Getting to the Domains Homepage
+    steps:
+
+    - instruction: Login to the Dashboard
+      image: '/assets/images/screenshots/draft2/login.png'
+
+    - instruction: Click the Side Bar menu button to the left of the page
+      image: /assets/images/mockdown/draft2/domains/open-the-sidebar.png
+
+    - instruction: Locate the Settings dropdown menu inside the side bar menu to the left of the page
```

Comment:
Tiny tweak, that's a non-issue: On line 16, "Side Bar" is capitalized, but on line 19 it is not. I would suggest making both (lines 16 and 19) capitalized or lower case to keep the docs consistent.